apps/theming-designer/src/components/Header.tsx

```diff
@@ -65,64 +87,51 @@ export class Header extends React.Component<IHeaderProps, IHeaderState> {
     return (
-      <Card
-        styles={{
-          root: { backgroundColor: 'white', minWidth: '99%', height: '35px', position: 'fixed', left: '0', top: '0', zIndex: 600 }
-        }}
-      >
-        <Stack horizontal tokens={{ childrenGap: 1200 }}>
-          <Stack horizontal>
-            <a href="https://www.microsoft.com" title="Microsoft Home Page" aria-label="Microsoft Home Page" className={microsoftLogo}>
-              <img src="https://themingdesigner.blob.core.windows.net/$web/MicrosoftLogo.png" className={microsoftLogo} />
-            </a>
-            <a
-              href="https://www.aka.ms/themedesigner"
-              title="Microsoft Theme Designer page"
-              aria-label="Microsoft Fabric Theme Designer page"
-              className={pipeFabricClassName}
-            >
-              <span> | UI Fabric Theme Designer </span>
-            </a>
-          </Stack>
-          <Stack horizontal styles={{ root: { position: 'absolute', right: '25px' } }}>
-            <PrimaryButton text="Export theme" onClick={this.showPanel} styles={{ root: { width: '130px', height: '35px' } }} />
-            <Panel
-              isOpen={this.state.showPanel}
-              type={PanelType.smallFixedFar}
-              onDismiss={this.hidePanel}
-              headerText="Export theme"
-              closeButtonAriaLabel="Close"
-              onRenderFooterContent={this.onRenderFooterContent}
-            >
-              <span>
-                <p>
-                  This code block initializes the theme you have configured above and loads it using the loadTheme utility function. Calling
-                  loadTheme will automatically apply the configured theming to any Fabric controls used within the same app. You can also
-                  export this example to CodePen with a few component examples below.
-                </p>
-              </span>
-              <div className={outputPanelClassName}>
-                <Pivot>
-                  <PivotItem headerText="Code">
-                    <textarea
-                      className={textAreaClassName}
-                      readOnly={true}
-                      spellCheck={false}
-                      value={codeHeader + this.state.themeAsCode}
-                    />
-                  </PivotItem>
-                  <PivotItem headerText="JSON">
-                    <textarea className={textAreaClassName} readOnly={true} spellCheck={false} value={this.state.jsonTheme} />
-                  </PivotItem>
-                  <PivotItem headerText="PowerShell">
-                    <textarea className={textAreaClassName} readOnly={true} spellCheck={false} value={this.state.powershellTheme} />
-                  </PivotItem>
-                </Pivot>
-              </div>
-            </Panel>
-          </Stack>
+      <Stack horizontal verticalAlign="center" grow={0} styles={headerStackStyles}>
+        <Stack horizontal grow={1} verticalAlign="center">
```

Comment:
Do we need 2 nested stacks? There is a need to grow an area, but that feels like an abuse of use here just to get flex-grow around an area.

I built this here: https://codesandbox.io/s/w2wjrw28pw

It uses this markup:

```tsx
  <div {...p}>
    <HeaderLogoImage src="https://themingdesigner.blob.core.windows.net/$web/MicrosoftLogo.png" />
    <HeaderDivider />
    <HeaderTitle>UI Fabric Theme Designer</HeaderTitle>
    <HeaderFar>
      <DefaultButton text="Export theme" />
    </HeaderFar>
  </div>
```

Reply:
I either need a stack, or a div with flex styles on it. I can't put grow on the link or buttons themselves, it has to go on a container. And the wrapper of all 3 links needs to be a stack as well. So I don't see a way past 2 stacks.

Reply:
in your example, does the div have flex styles assigned to it?